.goreleaser.yml

```diff
@@ -63,5 +63,14 @@ changelog:
     exclude:
       - "^.*Update docs.*$"
 release:
+  prerelease: auto
+  mode: prepend
```

Comment:
I'm hoping this will prepend new releases with older releases creating one large list. (not sure what it will look like).